INSTALL_STORE_CONTAINER.md

```diff
@@ -65,27 +65,13 @@ The container installation method requires Docker 1.10+ running on 64-bit linux.
 
 The node container (or plugin) should be installed on each Docker node where you want to consume StorageOS volumes or to present capacity to other nodes.
 
-### Network Block Device (NBD)
+### Linux-IO (LIO)
 
-(Optional) NBD is a default Linux kernel module that allows block devices to be run in userspace. Enabling NBD is recommended as it will increase performance for some workloads. To enable the module and increase the number of allowable devices, you must either run:
+StorageOS uses FUSE known as a Filesystem in Userspace. It is a software interface for Unix-like computer operating systems that lets non-privileged users create their own file systems without editing kernel code. In addition, StorageOS is developed using Linux-IO (LIO) Target. An open-source implementation of the SCSI target.
```

Comment:
I'd talk more about LIO here as that is what the user will see - fuse is pretty much hidden and could cause confusion.  Besides, we don't use it in the way other's are used to.

Reply:
fixed https://github.com/storageos/storageos.github.io/pull/148/commits/93a3652b57840070c18b0153615fe2568a9fcfb2